app/views/podcasts/show.html.erb

```diff
@@ -34,7 +34,7 @@
               </div>
             <% end %>
           <% else %>
-            <h2 class="text-muted text-center m-2"><%= t(".no_published") %></h2>
+            <p class="text-muted text-center m-2"><%= t(".no_published") %></p>
```

Comment:
This makes the "no published episodes" message consistent with the "No schedule Episodes" message.